Source/utils/language.cpp

```diff
@@ -279,6 +280,12 @@ bool HasTranslation(const std::string &locale)
 	return false;
 }
 
+bool IsLargeSmallFontLanguage()
+{
+	string_view code(sgOptions.Language.szCode, 2);
+	return code == "zh" || code == "ja";

```

Comment:
While Korean is not as badly affected I think it would still be best to apply this to it as well:

![image](https://user-images.githubusercontent.com/204594/142704559-002be816-6562-4c53-a498-73180fe55580.png)